### ISSUE TITLE
Fix lightbox gallery thumbnails that only say 'link' #254

### ISSUE THREAD

**Issue by erinchambers** [CDRH/orchid#339]
In VoiceOver, fancybox gallery is fully keyboard navigable and mostly screen reader friendly; the exception is that when the screen reader gets to the gallery of thumbnails (which pops in on the right), it reads each of them as "link". (So it goes "link, link, link, link," as you tab through.)

It would be best if these said "View image 1, View image 2, View image 3," etc. We decided in Teams on 2025-12-02 that we should write our own tweak/override for fancybox to solve this problem. If we can use some logic to generate the #, giving the link a title of "View image {#}" should be sufficient. (Just FYI, the images do not need alt text.)

# Suggested Fixes

1. Add a `title` attribute to the link, like so: `<a href="..." title="View image 1">`

OR

2. Add link text that is visually hidden (but available to screen readers) using the `sr-only` class, like so: `<a href="..."><span class="sr-only">View image 1</span></a>`

**Comment by wkdewey** [comment]
I think this is not handled in Orchid, but in individual sites. For instance, the fancybox code for SoH is in that project's repo, and I couldn't find anything in Orchid. I remember doing fancybox code for Whitman too.
Thumbnail code seems to be handled within the fancybox.min.js file.
If we want to hack into js, it might be better to have a non-minified file. Unfortunately I couldn't find old versions available for download, I think fancybox is only available paid.

**Comment by erinchambers** [comment]
I'll make a note to check each individual site for this; as Will mentioned, the accessibility problem described here is actually set up on individual sites, not through the Orchid base code.
